android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UrlDeviceDiscoveryService.java

```diff
@@ -493,7 +494,7 @@ public void restartScan() {
   }
 
   public boolean hasResults() {
-    return !mPwCollection.getGroupedPwPairsSortedByRank().isEmpty();
+    return !mPwCollection.getGroupedPwPairsSortedByRank(Utils.getDistanceComparator()).isEmpty();
```

Comment:
What is this used for?  Also, don't we have a getGroupedPwPairs that doesn't sort?